packages/protocol/contracts/governance/Governance.sol

```diff
@@ -838,6 +922,37 @@ contract Governance is IGovernance, Ownable, Initializable, ReentrancyGuard, Usi
     return voters[account].mostRecentReferendumProposal;
   }
 
+  /**
+   * @notice Checks if a byzantine quorum of validators has whitelisted the given hotfix.
+   * @param txHash The abi encoded keccak256 hash of the hotfix transaction.
+   * @return Whether validator whitelist tally >= validator byztanine quorum (2f+1)
+   */
+  function isHotfixPassing(bytes32 txHash) public view returns (bool) {
+    address[] memory validatorSet = getValidators().getRegisteredValidators();
+
+    uint256 tally = 0;
+    for (uint256 idx = 0; idx < validatorSet.length; idx++) {
+      if (hotfixes[txHash].whitelisted[validatorSet[idx]]) {
+        tally = tally.add(1);
+      }
+    }
+
+    return tally >= getValidators().getRegisteredValidatorsByzantineQuorum();
```

Comment:
I would put this in `UsingPrecompiles`

Reply:
didn't want to make UsingPrecompiles import SafeMath so I am leaving it in Governance (where other quorums live)